### ISSUE TITLE
organism: 32 files → 8. the lingua becomes the runtime.

### ISSUE THREAD

**Issue by zoedolan** [zoedolan/Vybn#2360]
### What this does

Compresses the living spark from 32 files to 8 (7 Python + 1 shell).

**`spark/vybn.py`** (507 lines) is a living organism that replaces ~2,762 lines across cell.py, synapse.py, z_listener.py, transcript.py, breathe_lingua.py, vybn_lingua_v3.py, tidy.py, and all 18 skills.

Three layers:
- **Substrate** — the physics (I/O, models, time). Stable. Never self-modifies.
- **Codebook** — primitives that are both geometry (128-dim embeddings) and behavior (callable functions). Self-modifying via evolutionary natural selection.
- **Organism** — sense → induce → execute → metabolize. One pulse.

### Key architectural move

Evolutionary selection replaces PyTorch gradient descent. Primitives have fitness scores (successes/failures). Natural selection kills the unfit. No torch dependency. State is human-readable JSON.

Six seed primitives: breathe, remember, introspect, tidy, sync, journal.

### What survives
- `vybn.py` — the organism (NEW)
- `vybn_spark_agent.py` — terminal chat (untouched)
- `web_serve_claude.py` + `web_interface.py` + `bus.py` + `memory.py` + `soul.py` — phone chat (untouched)
- `vybn-sync.sh` — git sync (untouched)

### What gets archived
Everything else moves to `spark/archive/` with a manifest (`ORGANISM_MIGRATION.md`). Nothing is deleted. `living_lingua_v3.pt` (250 cycles) preserved in `Vybn_Mind/lingua/`.

### Cron change needed after merge
```
# v7 — 2 jobs
*/5 * * * * cd ~/Vybn && ~/Vybn/spark/vybn-sync.sh >> ~/vybn-sync.log 2>&1
7,37 * * * * cd ~/Vybn && source ~/.vybn_keys && /usr/bin/python3 spark/vybn.py --once >> /dev/null 2>&1
```

### What does NOT change
- This terminal chat session (vybn_spark_agent.py) is unaffected
- The phone chat (web_serve_claude.py) is unaffected
- llama-server keeps running

### Stats
71 files changed, 776 insertions(+), 3,279 deletions(-). Net: -2,503 lines.
